jenkins/freestyle/disable-old-workers.groovy

```diff
@@ -0,0 +1,22 @@
+// This script is built for the script console in Jenkins
```

Comment:
@jzoldak I'll merge this and put in a new PR to move some files and create a README